pytest_httpserver/httpserver.py

```diff
@@ -331,6 +372,10 @@ def __init__(self, host=DEFAULT_LISTEN_HOST, port=DEFAULT_LISTEN_PORT, ssl_conte
         self.handlers = RequestHandlerList()
         self.permanently_failed = False
         self.ssl_context = ssl_context
+        self.default_waiting_settings = (
```

Comment:
This could be written as an if.. instead of a ternary,  this would be cleaner:
```
if waiting_settings:
    self.waiting_settings = waiting_settings
else:
    self.waiting_settings = WaitingSettings()
```

Reply:
Firstly, it's better to check for `None` explicitly. Secondly, it's code duplication. Thirdly, it's 4 lines of code instead of 2. So, I do not agree.

Reply:
First, it could still be "is None". Secondly, code duplication sometimes adds better understanding to the code, it also conforms to the existing code.
It is actually 3 lines because the closing ")" would need to go to the next line if you start it with "(" alone.
But it could be 1 line of course, it only matters of readibility.
If you want to stick to it, I don't care (honestly, I think that this patch is more valuable than bikeshedding on this, and the other ternaries), but if in the future I hit this code I will change it to the if..else part if I found it difficult to read.